triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -520,7 +523,10 @@ trait AbstractTriggerServiceTest extends AbstractTriggerServiceTestHelper {
     } yield succeed
   }
 
-  it should "restart trigger on run-time failure due to dropped connection" taggedAs availabilitySecurity inClaims withTriggerService(
+  it should "restart trigger on run-time failure due to dropped connection" taggedAs availabilitySecurity
```

Comment:
Same here. Please double-check all usages of `availabilitySecurity`.

Reply:
Likewise.